Assets/UPM/Console/Command.cs

```diff
@@ -4,9 +4,13 @@
 using System.Reflection;
 using UnityEngine;
 
-
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
 public class CommandAttribute : System.Attribute
 {
+    public string CustomName = ""; // maybe we wanna make a command named something that already exists, we can just add a custom name and use that instead :)
+                              // like this [Command(CustomName = "NameMe")]
+
+    public bool OnlyIngame = false; // maybe we make it so commands marked with this can only be run while the game is playing
```

Comment:
Not sure about this. The Editor folder can be used to filter out commands from the published game, but can you give an example of an In Game Only command? What value is added by adding this restriction?